src/libraries/System.Net.Http/tests/FunctionalTests/Socks/SocksProxyTest.cs

```diff
@@ -0,0 +1,40 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Net.Test.Common;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.Http.Functional.Tests.Socks
+{
+    [ConditionalClass(typeof(PlatformDetection), nameof(PlatformDetection.IsNotBrowser))]
+    public sealed class SocksProxyTest : HttpClientHandlerTestBase
+    {
+        public SocksProxyTest(ITestOutputHelper helper) : base(helper)
+        {
+        }
+
+        [Theory]
+        [InlineData("socks4")]
+        [InlineData("socks4a")]
+        [InlineData("socks5")]
+        public async Task TestLoopbackHttp1Async(string schema)
```

Comment:
VS can discover the test, but can't run it, local dotnet test can't discover it. What's wrong?

Reply:
OK I see they are green in CI